api/internal/plugins/builtinconfig/transformerconfig.go

```diff
@@ -114,12 +117,18 @@ func (t *TransformerConfig) AddSuffixFieldSpec(fs types.FieldSpec) (err error) {
 	return err
 }
 
-// AddLabelFieldSpec adds a FieldSpec to CommonLabels
-func (t *TransformerConfig) AddLabelFieldSpec(fs types.FieldSpec) (err error) {
+// AddCommonLabelFieldSpec adds a FieldSpec to CommonLabels
+func (t *TransformerConfig) AddCommonLabelFieldSpec(fs types.FieldSpec) (err error) {
 	t.CommonLabels, err = t.CommonLabels.MergeOne(fs)
 	return err
 }
 
+// AddLabelFieldSpec adds a FieldSpec to Labels
+func (t *TransformerConfig) AddLabelFieldSpec(fs types.FieldSpec) (err error) {
```

Comment:
Maybe it is better to use `CommonLabels` and `Labels` for the function name.
I'll leave it to your decision.

Reply:
Makes sense to me, will rename!

Reply:
Renamed!